ts/deck-options/DateInput.svelte

```diff
@@ -0,0 +1,30 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script>
+    import Col from "../components/Col.svelte";
+    import ConfigInput from "../components/ConfigInput.svelte";
+    import Row from "../components/Row.svelte";
+
+    export let date;
+</script>
+
+<div>
+    <ConfigInput>
+        <Row --cols={2}>
+            <Col>
+                <slot />
+            </Col>
+            <Col>
+                <input bind:value={date} type="date" />
```

Comment:
This doesn't look great on Safari, especially when not set (will attach photo in follow-up). You can test with `ANKI_API_HOST=0.0.0.0 ./run`, then surfing to `http://localhost:40000/_anki/pages/deck-options.html#test` on a mobile device.

Reply:
![IMG_1948](https://github.com/ankitects/anki/assets/180542/b5ecb8a4-4601-41ac-b987-2ecf5f0656c6)

Reply:
I'm afraid I can't test this as I don't have an Iphone :(

Reply:
perhaps it could be just a text input accepting a simple format like today is 2024-01-29

Reply:
Text input is too prone to user error